src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -34,6 +37,22 @@ public override UIView HitTest(CGPoint point, UIEvent? uievent)
 				return null!;
 			}
 
+			if (!result.UserInteractionEnabled)
+			{
+				// If the child also has user interaction disabled (IOW the child is InputTransparent),
+				// then we also want to exclude it from the hit testing.
+
+				return null!;
+			}
+
+			if (result is LayoutView layoutView && !layoutView.UserInteractionEnabledOverride)
+			{
+				// If the child is a layout then we need to check the UserInteractionEnabledOverride
+				// since layouts always have user interaction enabled.
+
+				return null!;
+			}
```

Comment:
And we need to ensure that transparent views/layouts also allow taps to pass through.